### PR TITLE
console log

### PR DESCRIPTION
**Pull request by VisheshSaluja**
Issue#550
fixed for processes individually - process.js

fixing for process in workflow - process.sidepanel.js

### CODE REVIEW

**Comment by VisheshSaluja** [comment]
### How to Test:

1. **Create a Workflow:**
   - Add two processes.

2. **Example Code for the Processes:**

   ```python
   import time

   def process_1():
       while True:
           print("Process 1: Still running...")
           time.sleep(2)  # Simulate work

   if __name__ == "__main__":
       process_1()


### Test

- Run a process.
- Check if the output of the first process(while in execution) is being rendered in the console log of second process or not.
- Also make some other checks, which might break something

File: src/main/resources/static/js/gw.ssh.js [hunk 1]
```diff
@@ -228,50 +194,39 @@ GW.ssh = {
...
+              }
+          } else {
+              // Handle regular process log output
+              this.echo(e.data);
+          }
+      } else {
+          // console.log("No display to output");
+      }
+  } catch (err) {
+      // console.log(err);
```
Comment (ZihengSun) [inline src/main/resources/static/js/gw.ssh.js:224]: remove these commented out code

File: src/main/resources/static/js/gw.ssh.js [hunk 2]
```diff
@@ -228,50 +194,39 @@ GW.ssh = {
 },
 
 
+ws_onmessage: function (e) {
```
Comment (ZihengSun) [inline src/main/resources/static/js/gw.ssh.js:197]: why is this function having the same name as the other function?

**Review by ZihengSun — CHANGES_REQUESTED** [review]
some comments

**Comment by VisheshSaluja** [comment]
will fix this and also found another bug, will fix that as well

**Pull request closed without merging**